spec/acceptance/sogo_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'spec_helper_acceptance'
+
+describe '::sogo' do
+  pp = <<-PUPPETCODE
+    $database = 'postgresql://sogo@127.0.0.1/sogo'
+
+    case $facts['os']['family'] {
+      'RedHat': {
+        $extra_packages = ['sope49-gdl1-postgresql']
+        if versioncmp($facts['os']['release']['major'], '7') >= 0 {
+          file_line { 'fix_sogo_systemd_unit':
+            ensure => 'absent',
+            path => '/usr/lib/systemd/system/sogod.service',
+            line => 'PIDFile=/var/run/sogo/sogo.pid',
+            notify => Service['sogod'],
+          }
+	}
+      }
+      'Debian': {
+	$extra_packages = ['sope4.9-gdl1-postgresql']
```

Comment:
can you indent this with two spaces as well, please?

Reply:
Fixed, a lost tab made it look good within vi until I fixed my trailing whitespace highlighter :-D